container/raw/handler.go

```diff
@@ -112,6 +117,16 @@ func newRawContainerHandler(name string, cgroupSubsystems *libcontainer.CgroupSu
 		}
 	}
 
+	if !isRootCgroup(name) {
+		// collect network metrics only for root cgroup
+		ignoreMetrics.Add(container.NetworkUsageMetrics)
```

Comment:
I think you need to make a deep copy of ignoreMetrics, or it will affect all containers.